test/rules/no-irregular-whitespaces/test.builder.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { IRREGULAR_WHITESPACE } from '../../../src/rules/noIrregularWhitespacesRule';
+
+const matches = IRREGULAR_WHITESPACE.source
+    .substring(1, IRREGULAR_WHITESPACE.source.length - 2)
+    .match(/\\\w+/g)
+    .filter(match => ['\\u2029', '\\u2028'].indexOf(match) === -1) // those seems to break the parser
+    .map(match => JSON.parse(`"${match}"`) as string);
+
+function writeFile(path: string, content: string) {
+    fs.unlink(path, () => {
```

Comment:
why do you need to unlink?

also, use the synchronous versions of these functions. we don't gain anything from async here.